app/scripts/lizard/views/DdscTimeseries.js

```diff
@@ -356,19 +360,41 @@ var mockingjay = {
 
 Lizard.geo.Popups.DdscTimeseries = {
   getPopupContent: function (location, region) {
+
+    var doneCb = function () {
+      var popupView = new Lizard.Views.LocationPopup({
+        collection: tsCollection
+      });
+      var popupContent = popupView.render();
+      region.show(popupContent);
+      Lizard.App.vent.trigger('ResizePopup');
```

Comment:
No: http://marionettejs.com/docs/marionette.application.html#event-aggregator and this code was only encapsulated in `doneCb` anyway.